### PR TITLE
Add an initial TravisCI configuration

### PR DESCRIPTION
**Pull request by zianc**
This PR adds an initial TravisCI configuration.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+dist: xenial
+
+language: node_js
+
+node_js:
+  - node
```
Comment (razvanm) [inline .travis.yml:6]: Is there a way to pick a more specific version?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+install:
+  - npm install web
+
+script:
+  - npm ci --prefix=./web/
+  - npm run --prefix=./web/ lint
+  - npm ci --prefix=./web/
+  - npm run --prefix=./web/ test -- --watch=false
+  - npm ci --prefix=./web/
+  - npm run --prefix=./web/ e2e
```
Comment (razvanm) [inline .travis.yml:22]: Can we do a `cd  web` at the beginning and avoid passing `--prefix=./web/`? If this are run in a single shell session we can run the `npm ci` only at the beginning.

File: .travis.yml (hunk 1 above)
Comment (zianc) [reply .travis.yml:6]: `node` provides the latest stable version. Do we want a different version?

Comment (razvanm) [reply .travis.yml:6]: I think we should use what we get on our remote environment. That's why I picked 10.21 for the CircleCI. :-)

```
$ node --version
v10.21.0
```

Comment (zianc) [reply .travis.yml:6]: Makes sense! Will switch it over now

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+dist: xenial
+
+language: node_js
+
+node_js:
+  - 10.12.0
```
Comment (razvanm) [inline .travis.yml:6]: I think this needs to be `10.21.0`.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  - 10.21.0
+
+sudo: false
+
+addons:
+  - chrome: stable
+
+before_install: cd web
+install:
+  - npm install
```
Comment (razvanm) [inline .travis.yml:15]: Don't we need to use `npm ci` in here and drop the one from the `script`?

File: web/package.json [hunk 5]
```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e --webdriverUpdate=false",
+    "postinstall": "node_modules/.bin/webdriver-manager update --gecko false --standalone false --versions.chrome=84.0.4147.89"
```
Comment (razvanm) [inline web/package.json:11]: Can you roll back the changes to this line? I think not doing that will cause the merge to fail.

**Pull request merged**